### DISCUSSION TITLE
Screen orientation

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#483] [Q&A] [answered]
Following problem:
I use the CRUISER with the motorcycle (landscape format) and with the convertible (portrait format).
If I now use different routing profiles when planning, do I have to switch all the profiles used from landscape to portrait format each time?
Couldn't this portrait/landscape format thing be set across profiles?
The system setting does not work in curves. The smartphone plays washing machine.

**Comment by devemux86** [comment]
Do you mean to make the "Screen orientation" and "Screen orientation mode" global settings?

**Reply by devemux86** [reply]
However, screen orientation usually is different in cars, motorcycles, bikes, hands, etc.

Why not create 2 app profiles for the motorcycle and the car with different orientation?
Can use the proper profile in each vehicle without need to change any setting.

**Reply by Leeb64** [reply]
How about that:
All profiles have a selector switch-globalized settings/separate settings.
In this way you could switch all profiles the same except for the few individual ones...
Would save a lot of adjustment work...

**Reply by devemux86** [reply]
Cruiser has a large list of  settings (that we often change), so this is not simple.

Also the report is still not very clear: 2 profiles for 2 vehicles are expected to support different orientations.
This is already available, so where exactly is the need for something else?

**Reply by Leeb64** [reply]
Not 2 profiles but all the profiles that you use for the route.
Fast/Fast+Curvy/Curvy/Extra Curvy
Or am I not understanding something?

**Reply by devemux86** [reply]
Isn't the question about the screen orientation and different application profiles (via 1st menu)?

The [new routing profiles](https://github.com/devemux86/cruiser/discussions/254) are for route planning.

**Reply by Leeb64** [reply]
Generally, yes

**Reply by WolfgangZ42** [reply, marked as answer]
The screen orientation is for navigation (or follow mode) not for planning:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/b44d9772-bd76-4814-9ba4-d6cd1464de9d"/>

**Reply by devemux86** [reply]
@Leeb64 Now I think I understand what you mean.

App profiles (car, motorcycle, bike, walking, ...) contain all app settings, like screen orientation.
Route profiles (fastest, curvy, ...) just contain specific route options.

As @WolfgangZ42 mentioned [above](https://github.com/devemux86/cruiser/discussions/483#discussioncomment-9062380), to have fixed screen orientation in navigation,
just select only navigation in the screen orientation mode setting.
Then the route planning will continue to use the default device orientation (portrait or landscape).
